src/components/visualizations/utils/test/drilldownEventing.spec.tsx

```diff
@@ -506,4 +506,105 @@ describe("Drilldown Eventing", () => {
             expect(drillConfig.onFiredDrillEvent).toHaveBeenCalled();
         });
     });
+
+    describe("Drilling in Combo chart", () => {
+        const columnPoint: Partial<IHighchartsPointObject> = {
+            ...point,
+            series: { type: VisualizationTypes.COLUMN },
+        } as any;
+
+        const linePoint: Partial<IHighchartsPointObject> = {
+            ...point,
+            x: 2,
+            y: 3,
+            series: { type: VisualizationTypes.LINE },
+            drillIntersection: [
+                {
+                    id: "id4",
+                    title: "title4",
+                    header: {
+                        identifier: "identifier4",
+                        uri: "uri4",
+                    },
+                },
+                {
+                    id: "id5",
+                    title: "title5",
+                    header: {
+                        identifier: "identifier5",
+                        uri: "uri5",
+                    },
+                },
+                {
+                    id: "id6",
+                    title: "title6",
+                    header: {
+                        identifier: "identifier6",
+                        uri: "uri6",
+                    },
+                },
+            ],
+        } as any;
+
+        it("should return chart type for each point", () => {
+            const drillConfig: IDrillConfig = { afm, onFiredDrillEvent: jest.fn() };
+            const target: any = { dispatchEvent: jest.fn() };
```

Comment:
could it be `target: EventTarget`? and remove the `as EventTarget` below?

Reply:
Can't because it will throw the error at this line
```
const drillContext = target.dispatchEvent.mock.calls[0][0].detail.drillContext;
```
Since `mock` is not exist on `EventTarget` type